src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tBody = document.querySelector('tbody');
```

Comment:
Consider using `table.querySelector('tbody')` instead of `document.querySelector('tbody')` to ensure you're selecting the right element inside the table 🧐